doc/index.md

```diff
@@ -0,0 +1,148 @@
+# BeakerLib
+
+After fedorahosted.org got shut down this became the official new home for this project.
+
+BeakerLib is a shell-level integration testing library, providing
+convenient functions which simplify writing, running and analysis of
+integration and blackbox tests.
+
+:::{toctree}
+---
+maxdepth: 2
+titlesonly: true
+caption: Contents
+glob: true
+---
+manual
+roadmap
+:::
+
+## News
```

Comment:
I'm just wondering if this should not be moved / linked to elsewhere. Normally I create the release and the just refer to it so I do not need to touch the repo itself again.

Reply:
Sure, we can move it as needed. Are these created from the spec file changelog? Is there a way of extracting them and/or making them in a markdown format? I know packit can do the opposite.

Reply:
> No, it is a manual work.

Let me rephrase that. Is the intention to have it synchronized, or is there cases where one is more detailed than the other? Is one a subset of the other? E.g. I can implement a roadmap section, a news section and a changelog section, where only one of them is synchronized.

As for the format. Which one do you want as different pages?

Reply:
Hm, I'll try this way to maintain it. I may be better, after all. We can change it anytime in the future.

Reply:
Ok, np. What about

> As for the format. Which one do you want as different pages?

Reply:
I do not have any preference, I think the MD is fine. Just KISS works.

It could be populated as a part of the individual changes in some "upcoming" section and finalized with just before the actual release.

Reply:
Ok, I'll just quickly fix the `.gitignore`. Do feel free to request any changes, no matter how big or nitpicky they are. It is very likely that it is something that I could use for my own projects, and I tend to be more productive when I work on other people's stuff.